addons/l10n_in/models/account_invoice.py

```diff
@@ -67,6 +68,22 @@ def _compute_l10n_in_state_id(self):
             else:
                 move.l10n_in_state_id = False
 
+    @api.depends('partner_id', 'partner_shipping_id', 'company_id', 'l10n_in_state_id')
```

Comment:
```suggestion
    @api.depends('l10n_in_state_id')
```
That should be sufficient